### ISSUE TITLE
error

### ISSUE THREAD

**Issue by dok441** [Tommy228/TTT_Spectator_Deathmatch#86]
[ERROR] addons/spectator_deathmatch-master/lua/specdm_von.lua:801: bad argument #1 to 'error' (table expected, got string)
  1. error - [C]:-1
   2. serialize - addons/spectator_deathmatch-master/lua/specdm_von.lua:801
    3. func - addons/spectator_deathmatch-master/lua/sv_stats.lua:209
     4. unknown - lua/includes/extensions/net.lua:32

**Comment by MinIsMin** [comment]
Does this happen frequently or when doing something specific?
Putting
```
print(query_str)
print(query)
```
before [sv_stats.lua#L209](https://github.com/Tommy228/TTT_Spectator_Deathmatch/blob/master/lua/sv_stats.lua#L209) should help identifing the problem if it is somehow reproducible.
Note that every request is logged in console.

**Comment by dok441** [comment]
When you enter these commands in the server console, nothing happens.

**Comment by MinIsMin** [comment]
No, you need to put the text in the file [sv_stats.lua](https://github.com/Tommy228/TTT_Spectator_Deathmatch/blob/master/lua/sv_stats.lua#L209) before the line 209.

Did this error occur in normal TTT? If not, please test without any other addons in normal TTT.

**Comment by dok441** [comment]
It would be easy to experience it, knowing I'm the reason why this happened, if anything, is used ТТТ2 https://github.com/Alf21/TTT2 
Version of your work, sensitive changed, https://github.com/Alf21/TTT_Spectator_Deathmatch
It would be great if you implemented this fix into your version so that it works in this mode and you can continue to work on it.
